SConstruct

```diff
@@ -190,6 +190,9 @@ opts.Add(BoolVariable('builtin_squish', "Use the built-in squish library", True)
 opts.Add(BoolVariable('builtin_thekla_atlas', "Use the built-in thekla_altas library", True))
 opts.Add(BoolVariable('builtin_zlib', "Use the built-in zlib library", True))
 opts.Add(BoolVariable('builtin_zstd', "Use the built-in Zstd library", True))
+opts.Add(BoolVariable('builtin_hb', "Use the built-in HarfBuzz library", True))
+opts.Add(BoolVariable('builtin_icu', "Use the built-in ICU library", True))
+opts.Add(BoolVariable('use_sil_graphite2', "Use the external SIL Graphite library (LGPL licensed)", False))
```

Comment:
I don't see any LGPL text in `src/hb-graphite2.{cc,h}`, it has the same copyright header as HarfBuzz.